ipaserver/plugins/hbacrule.py

```diff
@@ -130,14 +115,15 @@ class hbacrule(LDAPObject):
     container_dn = api.env.container_hbac
     object_name = _('HBAC rule')
     object_name_plural = _('HBAC rules')
-    object_class = ['ipaassociation', 'ipahbacrule']
-    permission_filter_objectclasses = ['ipahbacrule']
+    object_class = ['ipaassociation']
```

Comment:
OK I realized you always set ipahbacrule or ipahbacrulev2, so this should be OK. But I really don't like so many magic places where switching of objectclass is happening